runtime/permissions/prompter.rs

```diff
@@ -274,8 +275,9 @@ impl PermissionPrompter for TtyPrompter {
     message: &str,
     name: &str,
     api_name: Option<&str>,
-    is_unary: bool,
+    mut is_unary: bool,
   ) -> PromptResponse {
+    is_unary = is_unary && !is_running_from_binary();
```

Comment:
Can you explain why this is changing unary permission depending on the fact if it's self-compiled binary? I don't understand this part

Reply:
I noticed that is_unary is set to true 'hardcoded'. For future use, I checked if is_unary was set to true or false. If true, I need to verify if it is running from binary by flag. If it's set to false, there's no need to check.

But I can unuse is _unary and depends on the flag, What do you want?